### DISCUSSION TITLE
how can  i use chat.js

### DISCUSSION THREAD

**Discussion by fedorkanan** [community/community#64881] [Programming Help]
### Body

i have problems using jart.js and the library doesn't have any effects on my site 

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by fedorkanan** [comment]
go to this website :
https://www.chartjs.org/docs/latest/
and you can see the roadmap 

w3school :
https://www.w3schools.com/ai/ai_chartjs.asp
you can learn how to use it 

their gitthub : 
https://github.com/chartjs
you can see the changes that they do

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
